pkgs/development/libraries/gupnp/1.6.nix

```diff
@@ -46,8 +47,8 @@ stdenv.mkDerivation rec {
   ];
 
   mesonFlags = [
-    "-Dgtk_doc=true"
-    "-Dintrospection=true"
+    "-Dgtk_doc=${lib.boolToString (stdenv.buildPlatform == stdenv.hostPlatform)}"
+    "-Dintrospection=${lib.boolToString (stdenv.buildPlatform == stdenv.hostPlatform)}"
```

Comment:
gobject-introspection actually works for cross compilation now, just put it in nativeBuildInputs. And if gtk_doc involves running binary for host platform, there is mesonEmulatorHook.